react/features/base/premeeting/components/web/ConnectionStatus.js

```diff
@@ -78,7 +160,7 @@ function ConnectionStatus({ connectionDetails, t, connectionType }: Props) {
     }, [ showDetails, toggleDetails ]);
 
     return (
-        <div className = 'con-status'>
+        <div className = { clsx(styles.root, 'con-status') }>
```

Comment:
Remove old SCSS class

Reply:
`_premeeting-screens.scss` was using `con-status` classs as a selector, I moved the styles related to it to the `ConnectionStatus` component.

Reply:
Oh, I missed that. I would've been ok to leave them, but thanks for going the extra mile on this